src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,163 @@
+package duke;
+
+import duke.task.Task;
+
+import java.util.ArrayList;
+
+/**
+ * User Interface class, print beautiful output to user
+ */
+public class Ui {
+    private final String lines = "____________________________" +
+            "________________________________";
+    private final String endline = "___________________________" +
+            "_________________________________\n";
```

Comment:
Variable name for constant should be in uppercase? i noticed similar issue in other areas